config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -14,6 +14,7 @@ presubmits:
       preset-service-account: "true"
     max_concurrency: 12
     name: pull-security-kubernetes-e2e-kops-aws
+    namespace: test-pods
```

Comment:
hmm. probably the security job generator should not write this field.

Reply:
can you add a line to this method to nil out the namespace? https://github.com/kubernetes/test-infra/blob/7c45489ea12f4b490ba1a890cda8beecc9d51a7e/config/jobs/kubernetes-security/genjobs.go#L123